deploy/terraform/main.tf

```diff
@@ -73,9 +73,9 @@ resource "null_resource" "tink_directory" {
 
   provisioner "remote-exec" {
     inline = [
-      "iptables -A FORWARD -i eth1 -o bond0 -j ACCEPT",
-      "iptables -A FORWARD -i bond0 -o eth1 -m state --state ESTABLISHED,RELATED -j ACCEPT",
-      "iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE",
+      "iptables -A FORWARD -i enp1s0f1 -o bond0 -j ACCEPT",
```

Comment:
I believe the machine type is an input the user can change, which could end up with different nic names. We already do nic name detection in setup.sh which I believe tf ends up calling can we make use of that info instead?

Reply:
What about just moving these commands into setup.sh entirely?

Reply:
sgtm, don't know why it was done here like this.

Reply:
Yeah, there's a spot in setup.sh where IP forwarding gets turned on. That's the right spot.